### ISSUE TITLE
GmsCore not working properly!

### ISSUE THREAD

**Issue by mintdrops** [microg/android_packages_apps_GmsCore#935]
I am not sure if I'm doing something wrong but all the apps I installed (from .apk sources) don't recongize the Play Services substitute. 

Micro G Selfcheck does not show any issues, all the relevant boxes are checked. I flashed Lineage with Micro G weeks ago. I had Lineage with OpenGApps before and only changed to Micro G so I could use all my pro version apps from the Google Store without installing Google on my device... Now my system is basically just Lineage OS with Micro G wasting space and resources.

If you have any idea how I can make GmsCore work in my apps, please tell me! 

Android 9, Lineage OS 16.0, OnePlus 5t, Magisk Installed, Root working

**Comment by Riamse** [comment]
Does your Lineage ROM support signature spoofing? You have to install from
here https://lineage.microg.org/

**Comment by mintdrops** [comment]
Yes the 3 boxes regarding signature spoofing are all checked in selfcheck. I installed it from the same source you pointed out.

**Comment by Nanolx** [comment]
Well, first of all you should tell us *what* is wrong, you haven't provided any information on what apps refuse to work, what happens to them, what error messages are shown and no logs.

"Not working" is nothing one could help you with.
